contents/blog/sunsetting-helm-support-posthog.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2023-01-05
+title: "Sunsetting Kubernetes support for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["tim-glaser"]
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
+---
+
+We're sunsetting support for our Kubernetes deployment for PostHog. Because we're an open source company, I want to be transparent about what this change means, and why we're making it. Around 3.5% of PostHog users currently use this type of deployment. We will continue to support the docker deployment of PostHog.
+
+## Why are we doing this?
+
+We realized that self-hosting a Kubernetes deployment was too difficult and burdensome for both our free users and our paying customers.
+
+When we launched our Kubernetes deploy, we were hoping we could get to a stage where we battle tested enough of the components. We hoped that we could automate enough to make self hosting and scaling PostHog seamless, with a minimum amount of effort.
```

Comment:
This is feedback on most of your suggestions, and apologies for this being direct: You've taken something written in my tone and changed it into something much more corporate feeling, and not how I'd write this. Do you think this announcement needs that veneer of corporate?

Reply:
Personal feeling is that this should be in Tim's tone as much as possible. If we want to do a more direct company version we can spin up an FAQ page and link to that. Which isn't a bad idea, but I'd like to just send people to one page if we can, rather than two.

Reply:
@timgl No need to apologize. They're suggestions for a reason and I think you're right. Feel free to resolve any suggestions you don't want to use. I suppose the only stuff we definitely need to add is details RE: SOC 2 / HIPAA?

@joethreepwood To your point, maybe we just have an FAQ at the bottom of Tim's post. Keeps everything on one page but also scannable for anyone who wants to skip to the action points?